build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+echo 'Generating prometheus_alerts.yaml'
+jsonnet  -S -e 'std.manifestYamlDoc((import "alert-object.libsonnet").prometheus.rules)' > prometheus_alerts.yaml
```

Comment:
can you document this in README file

Reply:
this is not the recommended prcedure. 
Are you sure about documenting it in `master`? Maybe we can add a doc in `extras`?